jenkins/docker/docker-re-release.jenkinsfile

```diff
@@ -0,0 +1,59 @@
+lib = library(identifier: 'jenkins@5.7.1', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/opensearch-project/opensearch-build-libraries.git',
+]))
+
+pipeline {
+    options {
+        timeout(time: 7, unit: 'HOURS')
+    }
+    agent none
+    parameters {
+        string(
+            name: 'PRODUCT',
+            description: "OpenSearch / OpenSearch-Dashboards",
+            trim: true
+        )
+        string(
+            name: 'TAG',
+            description: "Enter tag of the Product",
```

Comment:
It's just a tag Sayali, The input would be 1 or 2 for example

Reply:
Can you add an example in the description to be more clear? Also Does this take care to update latest and stuff? Also how many tags can you pass?

Reply:
Yes @gaiksaya, the latest is updated only when the corresponding version is latest, currently it's for 2.x. And it's completely taken care by the shared library. Currently tag and product are strings. I can change them to list. So we can pass multiple values at a time.

Reply:
@Divyaasm I would suggest use a choice param so people can choose the major version.

You did not give any example in the description and it will naturally confuse people.